### PR TITLE
feat: Implement new Langflow Logo throughout the application

### PR DESCRIPTION
**Pull request by mfortman11**
Updates the application to use the new logo

Screenshot 2024-11-11 at 1 00 43 PM
Screenshot 2024-11-11 at 1 00 29 PM

Screenshot 2024-11-11 at 12 58 25 PM
Screenshot 2024-11-11 at 1 00 11 PM

Screenshot 2024-11-11 at 12 58 11 PM
Screenshot 2024-11-11 at 1 00 02 PM

Favicon
Screenshot 2024-11-11 at 4 11 44 PM

### CODE REVIEW

**Review by Cristhianzl — APPROVED** [review]
lgtm

File: src/frontend/src/pages/SignUpPage/index.tsx [hunk 1]
```diff
@@ -85,13 +87,20 @@ export default function SignUp(): JSX.Element {
...
-      className="h-full w-full"
+      className="h-screen w-full"
     >
       <div className="flex h-full w-full flex-col items-center justify-center bg-muted">
         <div className="flex w-72 flex-col items-center justify-center gap-2">
-          <span className="mb-4 text-5xl">⛓️</span>
+          {ENABLE_NEW_LOGO ? (
+            <LangflowLogo
+              title="Langflow logo"
+              className="mb-4 h-[40px] w-[40px] scale-[1.5]"
```
Comment (Cristhianzl) [inline src/frontend/src/pages/SignUpPage/index.tsx:97]: can we change this to `h-10`?

File: src/frontend/src/pages/LoginPage/index.tsx [hunk 2]
```diff
@@ -65,7 +67,14 @@ export default function LoginPage(): JSX.Element {
     >
       <div className="flex h-full w-full flex-col items-center justify-center bg-muted">
         <div className="flex w-72 flex-col items-center justify-center gap-2">
-          <span className="mb-4 text-5xl">⛓️</span>
+          {ENABLE_NEW_LOGO ? (
+            <LangflowLogo
+              title="Langflow logo"
+              className="mb-4 h-[40px] w-[40px] scale-[1.5]"
```
Comment (Cristhianzl) [inline src/frontend/src/pages/LoginPage/index.tsx:73]: can we change this to `h-10 w-10`?

File: src/frontend/src/pages/DeleteAccountPage/index.tsx [hunk 3]
```diff
@@ -17,7 +19,14 @@ export default function DeleteAccountPage() {
   return (
     <div className="flex h-full w-full flex-col items-center justify-center bg-muted">
       <div className="flex w-72 flex-col items-center justify-center gap-2">
-        <span className="mb-4 text-5xl">⛓️</span>
+        {ENABLE_NEW_LOGO ? (
+          <LangflowLogo
+            title="Langflow logo"
+            className="mb-4 h-[40px] w-[40px] scale-[1.5]"
```
Comment (Cristhianzl) [inline src/frontend/src/pages/DeleteAccountPage/index.tsx:25]: can we change this to` h-10 w-10`?

File: src/frontend/src/pages/AdminPage/LoginPage/index.tsx [hunk 4]
```diff
@@ -49,7 +51,14 @@ export default function LoginAdminPage() {
   return (
     <div className="flex h-full w-full flex-col items-center justify-center bg-muted">
       <div className="flex w-72 flex-col items-center justify-center gap-2">
-        <span className="mb-4 text-5xl">⛓️</span>
+        {ENABLE_NEW_LOGO ? (
+          <LangflowLogo
+            title="Langflow logo"
+            className="h-[40px] w-[40px] scale-[1.5]"
```
Comment (Cristhianzl) [inline src/frontend/src/pages/AdminPage/LoginPage/index.tsx:57]: can we change this to` h-10 w-10`?

File: src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx [hunk 5]
```diff
@@ -144,11 +147,17 @@ export default function ChatView({
...
-              />
+              {ENABLE_NEW_LOGO ? (
+                <LangflowLogo
+                  title="Langflow logo"
+                  className="h-[40px] w-[40px] scale-[1.5]"
+                />
+              ) : (
+                <ChainLogo
+                  title="Langflow logo"
+                  className="h-[40px] w-[40px] scale-[1.5]"
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx:158]: can we change this to` h-10 w-10`?

File: src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx [hunk 6]
```diff
@@ -144,11 +147,17 @@ export default function ChatView({
...
             <div className="flex flex-col items-center justify-center gap-4 p-8">
-              <img
-                src="/src/assets/logo.svg"
-                alt="Chain logo"
-                className="h-[40px] w-[40px] scale-[1.5]"
-              />
+              {ENABLE_NEW_LOGO ? (
+                <LangflowLogo
+                  title="Langflow logo"
+                  className="h-[40px] w-[40px] scale-[1.5]"
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/newChatView.tsx:153]: can we change this to` h-10 w-10`?

**Review by Cristhianzl — CHANGES_REQUESTED** [review]
just some small adjustments

**Review by Cristhianzl — APPROVED** [review]
LGTM

File: src/frontend/index.html [hunk 7]
```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="icon" href="/favicon.ico" />
+    <link rel="icon" href="/favicon2.ico" />
```
Comment (mfortman11) [inline src/frontend/index.html:8]: named favicon2 for cache busting purposes can change back to favicon after a release or 2

File: src/frontend/src/components/appHeaderComponent/index.tsx [hunk 8]
```diff
@@ -57,14 +57,14 @@ export default function AppHeader(): JSX.Element {
...
+            <DataStaxLogo className="fill-black dark:fill-[white]" />
           ) : ENABLE_NEW_LOGO ? (
-            <ShortLangFlowIcon className="h-5 w-5 fill-black dark:fill-[white]" />
+            <LangflowLogo className="h-5 w-6" />
           ) : (
             <span className="fill-black text-2xl dark:fill-white">⛓️</span>
           )}
         </Button>
-        {!ENABLE_DATASTAX_LANGFLOW && (
+        {ENABLE_DATASTAX_LANGFLOW && (
```
Comment (mfortman11) [inline src/frontend/src/components/appHeaderComponent/index.tsx:67]: Corrects feature flag logic

**Review by anovazzi1 — APPROVED** [review]
lgtm

**Pull request merged**